src/components/Admin/Users/UsersList.tsx

```diff
@@ -122,7 +122,7 @@ class UsersList extends React.Component<UsersListProps, UsersListState> {
             }))
         }
         let roleUsers = await Promise.all(roleData);
-        // TS: BCP stopped here -- need to figure out what type find is returning!
+        // TS: @Jon -- need help to figure out what type find is returning!
```

Comment:
`Promise.all` takes an array of promises and returns an array with the result of resolving each of those promises (in the same order). So, in this case, it is an array, where each element looks like this `{'role': {'name': someName}, 'users': int[]}`